images/writing.md

```diff
@@ -7,8 +7,8 @@ Custom images allow you to define workspaces that include the dependencies,
 scripts, and user preferences helpful for your project.
 
 > If you're unfamiliar with how to create, build, and push Docker Images, please
-> review
-> [this tutorial by Docker](http://blog.shippable.com/build-a-docker-image-and-push-it-to-docker-hub)
+> review [this tutorial by
```

Comment:
the label implies that Docker Inc wrote this, but the post is by a different company?

The link also appears to be dead

Reply:
Updating to link to docs; unfortunately, the blog is having some CDN issues (I think) but we don't want to rely on them despite this